content/guides/server-sent-events/test/server_sent_events/service_test.clj

```diff
@@ -0,0 +1,39 @@
+(ns server-sent-events.service-test
+  (:require [clojure.test :refer :all]
+            [io.pedestal.test :refer :all]
+            [io.pedestal.http :as bootstrap]
+            [server-sent-events.service :as service]))
+
+(def service
+  (::bootstrap/service-fn (bootstrap/create-servlet service/service)))
+
+(deftest home-page-test
```

Comment:
These tests aren't very meaningful. You might want to get rid of them in the "Cleaning Up" section.